src/Symfony/Component/VarDumper/Test/VarDumperTestCase.php

```diff
@@ -29,7 +29,7 @@ public function assertDumpMatchesFormat($dump, $data, $message = '')
         $this->assertStringMatchesFormat(rtrim($dump), $this->getDump($data), $message);
     }
 
-    private function getDump($data)
+    public function getDump($data)
```

Comment:
@nicolas-grekas Wouldn't it be enough to have the method `protected` in both the class and the trait?